core/aws.py

```diff
@@ -0,0 +1,36 @@
+from io import FileIO
+
+from boto3 import client
+from botocore.exceptions import ClientError
+from flask import current_app
+
+from config import Config
+
+if hasattr(Config, "AWS_ACCESS_KEY_ID") and hasattr(Config, "AWS_SECRET_ACCESS_KEY"):
+    _S3_CLIENT = client(
+        "s3",
+        aws_access_key_id=Config.AWS_ACCESS_KEY_ID,
+        aws_secret_access_key=Config.AWS_SECRET_ACCESS_KEY,
+        region_name=Config.AWS_REGION,
+        endpoint_url=Config.AWS_ENDPOINT_OVERRIDE,
+    )
+else:
+    # boto gets access keys from the environment directly in AWS
+    _S3_CLIENT = client(
```

Comment:
Will this break things if people are running locally without the docker runner? Should we put a separate env var to enable this functionality?

Reply:
It won't break - an exception will just be caught and logged